client/ayon_core/hosts/maya/plugins/publish/extract_unreal_skeletalmesh_fbx.py

```diff
@@ -74,7 +74,7 @@ def process(self, instance):
             renamed_to_extract.append("|".join(node_path))
 
         with renamed(original_parent, parent_node):
-            self.log.debug("Extracting: {}".format(renamed_to_extract, path))
+            self.log.debug("Extracting: {}".format(renamed_to_extract))
```

Comment:
Can rely on `logging` formatting here.
```suggestion
            self.log.debug("Extracting: %s", renamed_to_extract)
```

Reply:
Sure should.

```python
logging.info("hello %s", ["test", "bar"])
# root : hello ['test', 'bar']
```

Reply:
But if you would modify the list, the publisher would not show the values that the variable had at the moment it was logged.

```python
my_var = ["test", "bar"]
logging.info("hello1 %s", my_var)
my_var.append("new")
logging.info("hello2 %s", my_var)
```
Both of these logs would show `hello ['test', 'bar', 'new']` in UI.

Reply:
Well - I haven't looked much into how the `logging` library works but I wouldn't expect that to be the case.

Also, the publisher UI handles the log directly with its handler, so then I definitely wouldn't expect that to be the case.

But hey, happy to be proven wrong.

Reply:
We actually did hit these issues when we used the logging formatting. If plugin changed content of logged data it did not show correct output in UI.

Reply:
Interesting - that's very good to know actually; I wasn't aware of that being a potential issue. I understand the mutable arguments, etc. but would have expected a library like `logging` to maybe keep around immutable copies or whatever.

Thanks.